larg4/pluginActions/ParticleListAction_service.cxx

```diff
@@ -338,9 +342,9 @@ namespace larg4 {
           // figure out the ultimate parentage of this particle
           // first add this track id and its parent to the fParentIDMap
           fParentIDMap[trackID] = parentID;
-
-          fCurrentTrackID = -1 * this->GetParentage(trackID);
-
+	  fCurrentTrackID = this->GetParentage(trackID);
```

Comment:
The -1 has been removed here but kept below on line 374.  Is that intentional?

Reply:
I got rid of the -1 everywhere it doesn't matter and ID always should be positive  (usually I enforce it by using unsigned int)